manager/state/raft/raft.go

```diff
@@ -474,8 +474,9 @@ func (n *Node) joinCluster(ctx context.Context) error {
 // raft node that can be modified and customized
 func DefaultNodeConfig() *raft.Config {
 	return &raft.Config{
-		HeartbeatTick:   1,
-		ElectionTick:    3,
+		HeartbeatTick: 1,
+		// Recommended value in etcd/raft is 10 x (HeartbeatTick).
```

Comment:
Does it make sense to have more of a description around why? It would certainly be useful to at least add that historically lower values have potentially caused frequent leader elections.

Reply:
done.